src/System.Net.Sockets/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -1483,6 +1487,9 @@ public void ReceiveFromV4BoundToSpecificV6_NotReceived()
             });
         }
 
+        // Binds to a specific port on 'connectTo' which on Unix may already be in use
+        // Test preserved if we change these tests to run in series (instead of parallel) to avoid port collision
+#if false
```

Comment:
The comment says the test was preserved in case we have a different mechanism to run these tests in series so they don't interfere with each other.

So I can either remove the code, or add an issue to run these tests in series.

Do you think the tests are important enough to preserve for future?